ranger/data/scope.sh

```diff
@@ -169,17 +169,6 @@ handle_image() {
         #              -- "${FILE_PATH}" "${IMAGE_CACHE_PATH%.*}" \
         #         && exit 6 || exit 1;;
 
-
-        ## ePub, MOBI, FB2 (using Calibre)
```

Comment:
I don't really see a reason to remove epub-thumbnailer, it's not used unless it's installed. Not everyone wants to install a relatively large package like calibre just for cover previews.

Reply:
> Let me know if there's more serious reasons not to include epub-thumbnailer.

Not really, I removed it because I'm the person who introduced it and I only did that because I didn't know how much better Calibre's format support is. Also, people who deal with e-books probably have Calibre installed, but the same isn't true of epub-thumbnailer. If you'd still like epub-thumbnailer support back, I can re-add it and squash the commits of course.

Reply:
I'm open to replacing it with another lightweight solution but I do think it's a net positive to have an alternative.